src/glwidget.cpp

```diff
@@ -1299,11 +1299,6 @@ void GLWidget::paintGL()
             zEbene += 1.0;
     }
 
-    if( zRot <= 0 )
```

Comment:
The fact that the immediately preceding loop is a `while(true) {...}` one strikes me as inelegant (or one that ought to be converted to a `do {...} while( ... );` one) - but that _is_ why this code is currently pointless, if I understand it correctly?